arm-templates/base/deploy-aks.json

```diff
@@ -125,6 +129,21 @@
                 },
                 "networkProfile": {
                     "networkPlugin": "azure",
+                    "loadBalancerSku": "[if(contains(resourceGroup().name, 'dev'), 'Standard', 'Basic')]", //Standard only for dev until cluster
```

Comment:
When recreating prod cluster we should switch over to standard SKU as multiple node pools are no longer supported for basic load balancer.

Load balancer profile is available only for standard SKU public IPs in Azure (currently only set for dev cluster)